packages/runtime-handler/src/dev-runtime/internal/crossForkLogger.ts

```diff
@@ -0,0 +1,35 @@
+import { LoggerInstance } from '../types';
+
+export class CrossForkLogger implements LoggerInstance {
+  constructor() {}
+
+  debug(msg: string) {
+    this.sendLog('debug', msg);
+  }
+
+  info(msg: string) {
+    this.sendLog('info', msg);
+  }
+
+  warn(msg: string, title: string = '') {
+    this.sendLog('warn', msg, title);
+  }
+
+  error(msg: string, title: string = '') {
+    this.sendLog('error', msg, title);
+  }
+
+  log(msg: string, level: number) {
+    this.sendLog('log', msg, level);
+  }
```

Comment:
curious, when do we use a [log level](https://github.com/twilio-labs/serverless-toolkit/blob/main/packages/runtime-handler/src/dev-runtime/types.ts#L72)? Also, what does `level` represent here?